ui/page/main_page.go

```diff
@@ -218,20 +231,51 @@ func (mp *MainPage) OnNavigatedTo() {
 			go mp.WL.MultiWallet.Politeia.Sync()
 		}
 	}
+
+	mp.updateExchangeSetting()
+
+	mp.UpdateBalance()
```

Comment:
Does fetching the balance take time? If it doesn't, this should be in `OnNavigatedTo` since it doesn't make sense to fetch the balance when the page isn't being displayed yet, while it makes sense to refresh the balance whenever the page is about to be displayed.

Reply:
it is currently in the `OnNavigatedTo()`

Reply:
Then we certainly don't need it here. Any balance calculated here will not be displayed until `OnNavigatedTo` is called and that will re-calculate the balance.

Reply:
From my understanding, want to be able to update the balance when the page loads, and when there is an update from a listener. 

Moving this from `OnNavigatedTo`, i am not sure where else to place it. You can review the update i did if it works.

Reply:
Oy my, missed the fact that is in `OnNavigatedTo`. Sorry, it's fine here.